tests/Feature/SearchTest.php

```diff
@@ -13,6 +13,8 @@ class SearchTest extends TestCase
     /** @test */
     public function a_user_can_search_threads()
     {
+        if ( ! config('scout.algolia.id')) return true;
```

Comment:
Shouldn't we mark the test as skipped?

Reply:
@m1guelpf How is that done? Phpunit does warn that the test didn't perform assertions.

Reply:
You can do `$this->markTestSkipped('Reason')`

Reply:
Awesome. Done.